libraries/AP_Math/location.cpp

```diff
@@ -46,6 +46,30 @@ float longitude_scale(const struct Location &loc)
 #endif
 }
 
+// Added for use by SoaringController
+// return distance to north
+float get_offset_north(const struct Location &loc1, const struct Location &loc2)
```

Comment:
You are correct indeed, these methods are a remnant of Ardusoar for Arduplane 3.2 and aren't necessary anymore. Removed them in PR# 5737 and replaced their functionality with location_diff.